Library/Formula/passpie.rb

```diff
@@ -0,0 +1,71 @@
+class Passpie < Formula
+  homepage "https://marcwebbie.github.io/passpie"
+  url "https://github.com/marcwebbie/passpie/archive/0.1.tar.gz"
+  sha256 "be25cffe99ecf19e7a239139ab5e2b8b1f3bb50e22431dd19a0e19ac31c59c49"
+  head "https://github.com/marcwebbie/passpie.git"
+
+  depends_on :python3
```

Comment:
If this can use Python 2.7, let's do
`depends_on :python if MacOS.version <= :snow_leopard`
instead.